feature/interface/singleton/otg_tests/singleton_test/README.md

```diff
@@ -73,6 +73,37 @@ a new testbed configuration with the desired port types.
 *   Ensure inbound and outbound unicast counters are the same
 *   Ensure counters increment at the selected SAMPLE interval
 
+### RT-5.1.4 [TODO: https://github.com/openconfig/featureprofiles/issues/2338]
+#### Breakout must be explicitly configured by gNMI client
+
+*   On DUT Port-1 with a QSFP-DD 400GBASE-DR4 transceiver inserted
+*   Ensure no breakout is configured
+*   Set Port-1 port-speed to 100G
+    *   /interfaces/interface/ethernet/config/port-speed
+*   Validate that the DUT does not create breakouts implicitly and does not set the breakout speed
+    *   /components/component/port/breakout-mode/groups/group/config
+    *   /components/component/port/breakout-mode/groups/group/config/index
+    *   /components/component/port/breakout-mode/groups/group/config/breakout-speed
+*   Validate the port-speed of the interface is not changed to 100G
```

Comment:
@rohit-rp @dplore thanks for the changes.

I still have concerns about this part.
If a mismatched speed is configured and accepted by a device, the device can have two options:
1) ignore the mismatching configuration, and set the speed based on the equipped transceiver, or
2) take into the account accepted config, and bring down the interface

I don't believe opt#1 should be the only option. If an operator configures an explicit statement that an interface should use 100G, he may mean that this interface _should_ have oper-status UP only with 100G speed transceiver.
But opt#1 implies that port-speed statement doesn't mean anything and just ignored - and all operations are based on dynamic port speed detection of the transceiver.

Reply:
Let's stick with option #2 only.  We had some discussion of this type of issue in the past where an interface config is mismatched with the installed transceiver.  Operationally we don't want to be blocked deploying a configuration due to a single port mismatch (or even many ports).    Therefore I think we always want option 2 and  never option 1.